t/perf/opcount.t

```diff
@@ -788,4 +788,13 @@ test_opcount(0, "builtin::is_tainted is replaced with direct opcode",
                     is_tainted => 1,
                 });
 
+# sassign + padsv combinations are replaced by padsv_store
+test_opcount(0, "sassign + padsv replaced by padsv_store",
+                sub { my $x = undef; my $y; my $z = $y = 3; },
+                {
+                    padsv    => 1,
+                    padsv_store  => 3,
+                    undef    => 1,
```

Comment:
Tiny whitespace alignment issue here. Make the longest one have a single space, and align the others at the `=>` to match.

Reply:
Thanks. I took out the `undef` assignment, since #20077 will change it, and aligned the spaces on the remainder.

Reply:
Re: perldelta.pod, an entry per optimization seems like too much. Would a placeholder that covers multiple (and could link to individual PRs or commits) suffice? Something like: 
"Multiple optree optimizations have been applied in order to speed up common operations."